packages/datatrak-web-server/src/app/createApp.ts

```diff
@@ -71,8 +71,29 @@ export function createApp() {
     .get<SingleSurveyResponseRequest>('surveyResponse/:id', handleWith(SingleSurveyResponseRoute))
     .use('signup', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
     // Forward everything else to central server
-    .use('*', forwardRequest(CENTRAL_API_URL, { authHandlerProvider }))
-    .build();
+    .use('*', forwardRequest(CENTRAL_API_URL, { authHandlerProvider }));
+
+  await builder.initialiseApiClient([
+    { entityCode: 'DL', permissionGroupName: 'Public' }, //	Demo Land
+    { entityCode: 'FJ', permissionGroupName: 'Public' }, //	Fiji
+    { entityCode: 'CK', permissionGroupName: 'Public' }, //	Cook Islands
+    { entityCode: 'PG', permissionGroupName: 'Public' }, //	Papua New Guinea
+    { entityCode: 'SB', permissionGroupName: 'Public' }, //	Solomon Islands
+    { entityCode: 'TK', permissionGroupName: 'Public' }, //	Tokelau
+    { entityCode: 'VE', permissionGroupName: 'Public' }, //	Venezuela
+    { entityCode: 'WS', permissionGroupName: 'Public' }, //	Samoa
+    { entityCode: 'KI', permissionGroupName: 'Public' }, //	Kiribati
+    { entityCode: 'TO', permissionGroupName: 'Public' }, //	Tonga
+    { entityCode: 'NG', permissionGroupName: 'Public' }, //	Nigeria
+    { entityCode: 'VU', permissionGroupName: 'Public' }, //	Vanuatu
+    { entityCode: 'AU', permissionGroupName: 'Public' }, //	Australia
+    { entityCode: 'PW', permissionGroupName: 'Public' }, //	Palau
+    { entityCode: 'TL', permissionGroupName: 'Public' }, //	Timor-Leste
```

Comment:
We do have some hardcoded logic in `web-config-server` somewhere that 'turns off' TL for users, which gets carried through to `datatrak-web` and `tupaia-web` as a result - not sure if we should remove TL from this list, just so that it isn't confusing for developers down the line?

Reply:
Ahh good catch! Will remove this